plugin/php/class-block-types.php

```diff
@@ -245,6 +245,7 @@ public function enqueue_block_editor_assets() {
 			'postTypes'                 => $post_types,
 			'doesRequireBackCompatList' => version_compare( get_bloginfo( 'version' ), '5.8', '<' ),
 			'canUseQueryLoop'           => version_compare( '5.8', get_bloginfo( 'version' ), '<=' ),
+			'fallBackImageCard'         => $this->plugin->asset_url( 'assets/images/placeholder-image-card.png' ),
```

Comment:
🥜 **Nitpick:** I'd prefer if we transform this image to either `webp` or at the very least `jpg`. I don't see any reason for this to be png and they are usually much larger that they need to be